randovania/gui/preset_settings/patcher_energy_tab.py

```diff
@@ -107,3 +124,7 @@ def _persist_progressive_damage(self, checked: bool):
     def _persist_dangerous_tank(self, checked: bool):
         with self._editor as editor:
             editor.set_configuration_field("dangerous_energy_tank", checked)
+
+    def _persist_ingame_difficulty(self, value: IngameDifficulty):
```

Comment:
If you ever feel bored, do it for all 😛. But it would be nice to at least make sure that changed lines / new lines etc. 
 in a PR define the types properly.